database_backup/database_backup.task.yaml

```
@@ -0,0 +1,29 @@
+slug: demo_database_backup
+name: Database backup
+description: "Shell task that runs a PostgreSQL database backup and uploads the backup to AWS S3."
+resources:
+  # Replace this with the slug of your DB resource.
+  db: demo_db
+
+parameters:
+  - name: File name
+    slug: file_name
+    type: shorttext
+    description: The name to be given to the .ZIP file that will be created. If no name is specified here, the current timestamp will be used.
+    required: false
+
+schedules:
+  midnight_daily:
+    cron: "0 0 * * *"
+
+shell:
+  entrypoint: database_backup.sh
+  envVars:
+    AWS_SECRET_ACCESS_KEY:
+      config: AWS_SECRET_ACCESS_KEY
+    AWS_ACCESS_KEY_ID:
+      config: AWS_ACCESS_KEY_ID
+    AWS_BUCKET:
+      value: test-airplane-bucket
+    AWS_REGION:
+      value: us-east-2
```

Let's configure this with the most common AWS region.

```suggestion
      value: us-east-1
```